tools/nbs.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import nbs_tools.extract_images
+import nbs_tools.extract_balls
```

Comment:
Pretty sure this file should be gone? Or have I forgotten to make a PR for that upgrade

Reply:
You forgot to make a PR for that upgrade? :P 
The ball extraction was added because a mechatronics student needed the data. 
I feel like it should be easier to extract data though? It seems strange to have to make a new tool for every type of message we want to decode to json.

Reply:
Yes, we re-created the ball extractor for the mecha student. Do I need to make any changes?

Reply:
That PR seems like it went in #291 so there shouldn't be an nbs.py but it might have been missed deleting this file.
Also why do you need a tool to extract things to json? There already exists the json tool

Reply:
It is just a custom tool for this mecha guy. The json tool is just a screen dump of everything. So do I need to do anything?

Reply:
If I recall correctly the JSON tool just dumps all recorded messages? He wanted a subset of the messages with specific data. But perhaps something like https://stedolan.github.io/jq/ could be used?

Reply:
At the very least, instead of making a new tool for every single thing you want to dump with JSON, add a command-line argument to the JSON dumper that makes it only dump out specific message types